# missing ; in source code causes build failure when configured via
# --without-X
diff -Naur R-2.8.0/src/main/platform.c R-2.8.0.new/src/main/platform.c
--- R-2.8.0/src/main/platform.c	2008-10-07 22:05:05.000000000 -0400
+++ R-2.8.0.new/src/main/platform.c	2008-10-31 08:44:10.000000000 -0400
@@ -1654,7 +1654,7 @@
 # ifdef HAVE_X11
     int X11 = NA_LOGICAL;
 # else
-    int X11 = FALSE
+    int X11 = FALSE;
 # endif
 #endif
 
